frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -87,11 +89,15 @@ export const surveysLogic = kea<surveysLogicType>([
         },
         updateSurveySuccess: () => {
             lemonToast.success('Survey updated')
+            actions.loadCurrentTeam()
         },
         setSurveysFilters: () => {
             actions.loadSurveys()
             actions.loadResponsesCount()
         },
+        loadSurveysSuccess: () => {
```

Comment:
`surveyLogic` calls `loadSurveys` when surveys are launched/updated/resumed, so this should cover team reloading for all those cases